WolvenKit.App/ViewModels/Documents/RedDocumentViewModel.cs

```diff
@@ -89,18 +89,41 @@ public RedDocumentViewModel(CR2WFile file, string path, AppViewModel appViewMode
 
         _path = path;
 
+        if (s_globalEditorDifficultyLevel == EditorDifficultyLevel.None)
+        {
+            s_globalEditorDifficultyLevel = settingsManager.DefaultEditorDifficultyLevel;
+        }
 
+        EditorDifficultyLevel = s_globalEditorDifficultyLevel;
+        GlobalEditorDifficultyLevelChanged += OnGlobalEditorDifficultyChanged;
+        
         _extension = Path.GetExtension(path) != "" ? Path.GetExtension(path)[1..] : "";
 
         Cr2wFile = file;
         IsReadOnly = isReadyOnly;
-        EditorDifficultyLevel = editorMode;
         _isInitialized = true;
         PopulateItems();
     }
 
+    private void OnGlobalEditorDifficultyChanged(object? sender, EventArgs e) =>
+        EditorDifficultyLevel = s_globalEditorDifficultyLevel;
+
     #region properties
 
+    private static event EventHandler? GlobalEditorDifficultyLevelChanged;
+
+    private static EditorDifficultyLevel s_globalEditorDifficultyLevel = EditorDifficultyLevel.None;
```

Comment:
rather than having static fields here, put them into the Settingsamanager

Reply:
you mean another hidden field that doesn't get serialized/read back? because the standard should be applied on first start

Reply:
yeah exactly. But I mean it could be serialized tbh Seems like something the user wants to have persist